script.js

```diff
@@ -284,6 +300,6 @@ function startLooping() {
       "After " + (rangeLines.length - 2) + " iterations.";
```

Comment:
You can now remove this rangeLines.length and use the stepNumber now as it will be more clear.